native-sql-engine/core/src/main/scala/com/intel/oap/expression/ConverterUtils.scala

```diff
@@ -504,6 +504,7 @@ object ConverterUtils extends Logging {
     case d: StringType =>
     case d: DateType =>
     case d: DecimalType =>
+    case d: TimestampType =>
```

Comment:
Will this pr depends on the previous pr?
https://github.com/oap-project/native-sql-engine/pull/209